Tests/DataParser/test_dataParser.py

```diff
@@ -0,0 +1,116 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+sys.path.append("../../DataParser")
+import filecmp
+import dataParser
+
+testFiles = "./testFiles"
+truth_folder = "./testFiles/truth_data"
```

Comment:
constants should be denoted by CAPITAL_LETTERS. Change the constants names here.